### ISSUE TITLE
Django 1.8 support

### ISSUE THREAD

**Issue by pyghassen** [bread-and-pepper/django-userena#491]
I was trying to use django-userena with Django 1.8, and added the tests to tox to handle it seems to fail, but I couldn't figure out what the problem is.

Here is the error:

Creating test database for alias 'default' ('file:memorydb_default?mode=memory&cache=shared')...
There is no South database module 'south.db.sqlite3' for your database. Please either choose a supported database, check for SOUTH_DATABASE_ADAPTER[S] settings, or remove South from INSTALLED_APPS.
ERROR: InvocationError: '/home/user/django-userena/.tox/py34-django18/bin/python userena/runtests/runtests.py userena umessages'

**Comment by johanndt** [comment]
Did you remove South from INSTALLED_APPS, since it's not used since Django 1.7+ anymore?

**Comment by nooperpudd** [comment]
So did I, Please remove `south` from the `umessage`
```
Migrated app 'umessages' contains South migrations. Make sure all numbered South migrations are deleted prior to creating Django migrations.
```

**Comment by swistakm** [comment]
Fixed and released as part of #505
